website/source/docs/agent/telemetry.html.md

```diff
@@ -769,6 +769,12 @@ These metrics are used to monitor the health of the Consul servers.
     <td>queries</td>
```

Comment:
> This increments when a server sends a (potentially blocking) RPC query.

That's pretty misleading both because of the "inside loop" issue above but also because we increment it at the _start_ of the query not "when the server sends". I think this is copy pasta from the client-side equivalent as it doesn't make much sense at all.

I'd suggest changing this to:

> This increments when a server receives a new blocking RPC request, indicating the _rate of new_ blocking query calls. See `consul.rpc.queries_blocking` for the current number of in-flight blocking RPC calls.

Reply:
👍